packages/selected-network-controller/src/SelectedNetworkController.ts

```diff
@@ -136,6 +143,48 @@ export class SelectedNetworkController extends BaseController<
     });
     this.#registerMessageHandlers();
 
+    this.messagingSystem
+      .call('PermissionController:getSubjectNames')
+      .filter(
+        (domain) =>
+          this.state.domains[domain] === undefined &&
+          this.state.domains[domain] === undefined,
```

Comment:
🤔 
```suggestion
          this.state.domains[domain] === undefined
```

Reply:
lol just gotta make sure it didnt become undefined since last time we checked